New: Indeed donation blog post (#621)
eslint/website: _posts/2019-09-09-indeed-donates-10000-to-eslint.md

---
layout: post
title: "Indeed donates $10,000 to ESLint"
tags:
  - Open Source
  - Funding
  - Open Collective
  - Sponsors
  - Backers
---
# Indeed donates $10,000 to ESLint

<p class="text-center">
    <a href="https://indeed.com/" title="Indeed" rel="noopener nofollow" target="_blank"><img class="lazyload" width="200" data-src="/assets/img/logos/indeed.svg" alt="Indeed" src="/assets/img/logos/indeed.svg"></a>
    <a href="https://eslint.org/" title="ESLint" target="_blank"><img class="lazyload" width="200" data-src="/assets/img/logo.svg" alt="ESLint" src="/assets/img/logo.svg"></a>
</p>

> We are happy to share that [Indeed](https://indeed.com) has donated $10,000 to the ESLint project as part of their [FOSS Contributor Fund](https://engineering.indeedblog.com/blog/2019/07/foss-fund-six-months-in/). The FOSS Contributor Fund is a unique idea, where all Indeed employees may nominate an open source project to receive a one-time donation and any employee who has contributed to an open source project during that month gets to vote on the nominees. We are honored to have been chosen as one of the projects that Indeed employees chose to support.

Reviewer: Nit: adding a comma, and changing a consecutive use of “chosen” and “chose”.

```suggestion
We are happy to share that [Indeed](https://indeed.com) has donated $10,000 to the ESLint project as part of their [FOSS Contributor Fund](https://engineering.indeedblog.com/blog/2019/07/foss-fund-six-months-in/). The FOSS Contributor Fund is a unique idea, where all Indeed employees may nominate an open source project to receive a one-time donation, and any employee who has contributed to an open source project during that month gets to vote on the nominees. We are honored to be one of the projects that Indeed employees chose to support.
```

I went with a small change, but an alternative phrasing for the last sentence could be something like “We are honored that Indeed employees chose to support ESLint.”
